### PR TITLE
Obj_Entotu

### PR DESCRIPTION
**Pull request by Kelebek1**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

### CODE REVIEW

File: src/overlays/actors/ovl_Obj_Entotu/z_obj_entotu.c [hunk 1]
```diff
@@ -28,23 +28,155 @@ const ActorInit Obj_Entotu_InitVars = {
...
+    { 0xFA, 0x7F, 0x14, 0x05, 0x01, 0x2D, 0x00, 0x00, 0xFF, 0x75, 0xF4, 0x69, 0x00, 0x00, 0x00, 0x00 },
+    { 0x00, 0x00, 0x08, 0xD5, 0x02, 0x5B, 0x00, 0x00, 0x02, 0x00, 0x03, 0xCC, 0x4D, 0x44, 0x34, 0xFF },
+    { 0x00, 0x00, 0x1B, 0xAC, 0x01, 0x2D, 0x00, 0x00, 0x02, 0x00, 0xE9, 0xD0, 0x00, 0x00, 0x00, 0x00 },
+    { 0xFB, 0xAD, 0x08, 0xD5, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x04, 0x00, 0x00, 0x00, 0x00, 0x00 },
+    { 0x05, 0x81, 0x14, 0x05, 0x01, 0x2D, 0x00, 0x00, 0x04, 0x8B, 0xF4, 0x69, 0x00, 0x00, 0x00, 0x00 },
+    { 0x04, 0x53, 0x08, 0xD5, 0x00, 0x00, 0x00, 0x00, 0x04, 0x00, 0x04, 0x00, 0x00, 0x00, 0x00, 0x00 },
+    { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x00, 0x10, 0x3A, 0x26, 0x21, 0x19, 0x7E },
+};
+
+s32 func_80A34700(s16 minutes) {
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_Obj_Entotu/z_obj_entotu.c:41]: This *might* be boolean, though I'm not completely convinced.

File: src/overlays/actors/ovl_Obj_Entotu/z_obj_entotu.c [hunk 2]
```diff
@@ -28,23 +28,155 @@ const ActorInit Obj_Entotu_InitVars = {
...
+
+        OPEN_DISPS(globalCtx->state.gfxCtx);
+
+        func_8012C28C(globalCtx->state.gfxCtx);
 
-extern UNK_TYPE D_06000158;
-extern UNK_TYPE D_06001C00;
+        gSPSegment(POLY_XLU_DISP++, 0x08,
+                   Gfx_TwoTexScroll(globalCtx->state.gfxCtx, 0, 0, sp57, 0x20, 0x20, 1, 0, sp56, 0x20, 0x20));
+        gSPSegment(POLY_XLU_DISP++, 0x09, Lib_SegmentedToVirtual(this->unk_148));
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_Obj_Entotu/z_obj_entotu.c:152]: Looking in the object, this is a Vtx.

File: src/overlays/actors/ovl_Obj_Entotu/z_obj_entotu.c [hunk 3]
```diff
@@ -28,23 +28,155 @@ const ActorInit Obj_Entotu_InitVars = {
...
+    { 0x00, 0x00, 0x1B, 0xAC, 0x01, 0x2D, 0x00, 0x00, 0x02, 0x00, 0xE9, 0xD0, 0x00, 0x00, 0x00, 0x00 },
+    { 0xFB, 0xAD, 0x08, 0xD5, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x04, 0x00, 0x00, 0x00, 0x00, 0x00 },
+    { 0x05, 0x81, 0x14, 0x05, 0x01, 0x2D, 0x00, 0x00, 0x04, 0x8B, 0xF4, 0x69, 0x00, 0x00, 0x00, 0x00 },
+    { 0x04, 0x53, 0x08, 0xD5, 0x00, 0x00, 0x00, 0x00, 0x04, 0x00, 0x04, 0x00, 0x00, 0x00, 0x00, 0x00 },
+    { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0x00, 0x10, 0x3A, 0x26, 0x21, 0x19, 0x7E },
+};
+
+s32 func_80A34700(s16 minutes) {
+    s32 ret = 0;
+    s16 time = gSaveContext.time * 0.021972656f;
```
Comment (EllipticEllipsis) [inline src/overlays/actors/ovl_Obj_Entotu/z_obj_entotu.c:43]: This is `TIME_TO_MINUTES_F` (Thanks @hensldm / @tom-overton ).

File: src/overlays/actors/ovl_Obj_Entotu/z_obj_entotu.c (hunk 1 above)
Comment (Kelebek1) [reply src/overlays/actors/ovl_Obj_Entotu/z_obj_entotu.c:41]: It can return 0, 1, 2 or 3.

Comment (EllipticEllipsis) [reply src/overlays/actors/ovl_Obj_Entotu/z_obj_entotu.c:41]: Oh so it can, I missed the other settings of ret.

**Review by EllipticEllipsis — APPROVED** [review]

**Review by hensldm — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**